networkdb/cluster.go

```diff
@@ -372,11 +374,20 @@ func (nDB *NetworkDB) gossip() {
 		networkNodes[nid] = nDB.networkNodes[nid]
 
 	}
+	printStats := time.Since(nDB.lastStatsTimestamp) >= nDB.config.StatsPrintPeriod
 	nDB.RUnlock()
 
+	if printStats {
+		healthScore := nDB.memberlist.GetHealthScore()
+		if healthScore == 0 {
+			logrus.Infof("NetworkDB healthscore:%d Node totally healthy", healthScore)
```

Comment:
The definition of totally healthy comes directly from memberlist: 
```
GetHealthScore gives this instance's idea of how well it is meeting the soft real-time requirements of the protocol. Lower numbers are better, and zero means "totally healthy".
```
Will change